### ISSUE TITLE
Set up new transcriber automated email

### ISSUE THREAD

**Issue by richpomfret** [FreeUKGen/FreeCENMigration#548]
Add in useful links and FAQ.

**Comment by Captainkirkdawson** [comment]
Why not use the tools already developed for REG???

**Comment by PatReynolds** [comment]
Use automation from FreeREG to send letter that Brenda currently sends, with links to the three documents she currently attaches.

**Comment by Captainkirkdawson** [comment]
Is this still a requirement?

**Comment by PatReynolds** [comment]
@DeniseColbert  can you reply to Kirk, please?

**Comment by DeniseColbert** [comment]
Apologies for the delayed reply; yes, this will be addressed in the Training work I'm doing with my FC Vol Coord hat on.

**Comment by Captainkirkdawson** [comment]
Is there a developer requirement?

**Comment by DeniseColbert** [comment]
Yes, a developer will be needed to add new/updated info and resources to the email.
